src/cls/rbd/cls_rbd.cc

```diff
@@ -4660,6 +4664,33 @@ std::string mirror_image_map_key(const string& global_image_id) {
   return MIRROR_IMAGE_MAP_KEY_PREFIX + global_image_id;
 }
 
+std::string group_key(const string &group_id) {
+  return GROUP_KEY_PREFIX + group_id;
+}
+
+std::string group_global_key(const string &global_id) {
+  return GROUP_GLOBAL_KEY_PREFIX + global_id;
+}
+
+std::string group_resync_key(const std::string& global_group_id,
+                             const std::string& group_name) {
+  return group_name + "_" + global_group_id;
```

Comment:
Since this is written to the header of the group being resynced, this information is not required. A simple "resync" key with a value set to true (or similar) should suffice.

Reply:
Separate CLS functions are probably not required for the group sync key either - perhaps  `metadata_set()` can be used instead (as is done by the image resync command)